src/components/Payment/PayPal/PayPal.tsx

```diff
@@ -35,11 +35,15 @@ const PayPal = ({
 
   const handleSubmit = () => {
     const checkbox = document.querySelector(
-      `.checkbox-paypal`
+      `#checkbox-paypal`
+    ) as HTMLInputElement;
+
+    const checkboxWrapper = document.querySelector(
```

Comment:
i think two things could be improved here:
1. let's not cast types. it takes away the help that TypeScript gives us
2. especially that you cast the type to be a `label` specifically, let's just look for the closest `label` element